GLMakie/src/screen.jl

```diff
@@ -302,23 +317,23 @@ function singleton_screen(debugging::Bool)
     return reopen!(screen)
 end
 
-const GLFW_FOCUS_ON_SHOW = 0x0002000C
-
 function Makie.apply_screen_config!(screen::Screen, config::ScreenConfig, scene::Scene, args...)
     apply_config!(screen, config)
 end
 
 function apply_config!(screen::Screen, config::ScreenConfig; start_renderloop::Bool=true)
     glw = screen.glscreen
     ShaderAbstractions.switch_context!(glw)
-    GLFW.SetWindowAttrib(glw, GLFW_FOCUS_ON_SHOW, config.focus_on_show)
+    GLFW.SetWindowAttrib(glw, GLFW.FOCUS_ON_SHOW, config.focus_on_show)
```

Comment:
```suggestion
    GLFW.SetWindowAttrib(glw, GLFW_FOCUS_ON_SHOW, config.focus_on_show)
```
- [ ] Revert this change (And the constant definition a few lines earlier)

This constant is not defined for the full compat range of `GLFW`. I thought I'd checked this in the past, but I must have been mistaken, and it's an orthogonal change anyway that doesn't need to happen with this PR.

Reply:
Actually, looking at JuliaGL/GLFW.jl#202, it looks like this PR is going to require GLFW >=3.3 since this PR is going to depend on the WindowContentScale functions that PR added. So instead of reverting the constant change, we need to bump the compat requirement.